app/react/kubernetes/ingresses/IngressDatatable/IngressDatatable.tsx

```diff
@@ -26,33 +34,60 @@ interface SelectedIngress {
 }
 const storageKey = 'ingressClassesNameSpace';
 
-const settingsStore = createPersistedStore(storageKey);
+const settingsStore = createStore(storageKey);
 
 export function IngressDatatable() {
+  const tableState = useTableState(settingsStore, storageKey);
   const environmentId = useEnvironmentId();
 
+  const { isAdmin } = useCurrentUser();
   const { data: namespaces, ...namespacesQuery } = useNamespaces(environmentId);
-  const ingressesQuery = useIngresses(
+  const { data: ingresses, ...ingressesQuery } = useIngresses(
     environmentId,
-    Object.keys(namespaces || {})
+    Object.keys(namespaces || {}),
+    {
+      autoRefreshRate: tableState.autoRefreshRate * 1000,
+    }
+  );
+
+  const filteredIngresses = useMemo(
+    () =>
+      ingresses?.filter(
+        (ingress) =>
+          (isAdmin && tableState.showSystemResources) ||
```

Comment:
does `showSystemResources` is bound to be an admin only? cluster admin can't view system resources?

Reply:
Nice spot! Cluster admin and portainer admin can view system resources - I'll update this here and in EE

Reply:
The same issue is in the services datatable, configmaps and secrets datatables too. I'll update these too. The logic should be `useAuthorizations('K8sAccessSystemNamespaces')`